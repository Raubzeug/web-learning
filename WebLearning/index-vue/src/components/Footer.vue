<template>
  <div class="footer-wrapper">
    <div class="footer">
      ПОДВАЛ
    </div>
  </div>
</template>

<script>
export default {
  name: "Footer"
};
</script>

<style lang="less" scoped>
@import "../library.less";
.footer-wrapper {
  .flex-column;
  width: 100%;
  position: fixed;
  bottom: 0;
  justify-content: center;
  align-items: center;
  margin-top: 20px;
  margin-bottom: 0;
  padding: 20px 0 10px 0;
  background-color: @base_color;
}
</style>
