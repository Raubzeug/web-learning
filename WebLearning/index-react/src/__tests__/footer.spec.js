import React from 'react'
import { create } from 'react-test-renderer'

import Footer from '../components/footer'

describe('footer component', () => {
    test('Matches the snapshot', () => {
        const footer = create(<Footer />)
        expect(footer.toJSON()).toMatchSnapshot()

    })
})