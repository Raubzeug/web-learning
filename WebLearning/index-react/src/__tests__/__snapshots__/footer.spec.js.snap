// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer component Matches the snapshot 1`] = `
<div
  className="footer-wrapper"
>
  <div
    className="footer"
  >
    ПОДВАЛ
  </div>
</div>
`;
